src/noisepy/seis/io/channelcatalog.py

```diff
@@ -3,11 +3,15 @@
 from functools import lru_cache
 
 import diskcache as dc
+import numpy as np
 import obspy
 import pandas as pd
 from datetimerange import DateTimeRange
 from obspy import UTCDateTime, read_inventory
 from obspy.clients.fdsn import Client
+from obspy.core.inventory import Channel as ch
+from obspy.core.inventory import Inventory, Network, Site
+from obspy.core.inventory import Station as st
```

Comment:
See my other comment below...
```suggestion
import obspy.core.inventory.Channel
from obspy.core.inventory import Inventory, Network, Site
import from obspy.core.inventory.Station
```